### ISSUE TITLE
BfA + ElvUI shows back extra widgets

### ISSUE THREAD

**Issue by mooreatv** [mooreatv/NeatMinimap#2]
![](https://cdn.discordapp.com/attachments/218957301111848962/621323647579062272/unknown.png)

reported by Yuyuli

**Comment by mooreatv** [comment]
also probably shouldn't hide bottom etc panels or at least have the option to keep them

**Comment by mooreatv** [comment]
![WoWScrnShot_091119_145408](https://user-images.githubusercontent.com/46745059/64738442-0c717d00-d4a4-11e9-848a-ecaebeb5ff28.jpg)

```
Bug Report for NeatMinimap vX.YY.ZZ
2019/09/11 14:50:30 -0700 @project-abbreviated-hash@

Bug report from slash command
Session messages log:
36.87 NeatMinimap DBG[1]: Localization not found for "Bug report from slash command"
NeatMinimap bug report open: Please submit on discord or on curse or github or email
36.87 NeatMinimap DBG[1]: Localization not found for "NeatMinimap bug report open: "
36.87 NeatMinimap DBG[1]: Localization not found for "Please submit on discord or on curse or github or email"
36.87 NeatMinimap DBG[1]: Got slash cmd: "bug"
92.85 NeatMinimap DBG[1]: Showing buttons - force nil
78.72 NeatMinimap DBG[1]: Hiding buttons - force true
78.72 NeatMinimap DBG[1]: Adding "LibDBIcon10_DetailsStreamer"
78.72 NeatMinimap DBG[1]: Adding "PixelPerfectAlignminimapButton"
78.72 NeatMinimap DBG[1]: Skipping "NeatMinimapFrame" function: 000001EAB3B6B9A0
78.72 NeatMinimap DBG[1]: Adding "MamaminimapButton"
78.72 NeatMinimap DBG[1]: Adding "LibDBIcon10_AHDBminimapButton"
78.72 NeatMinimap DBG[1]: Skipping "HelpOpenWebTicketButton" function: 000001EAB3B6DED0
78.72 NeatMinimap DBG[1]: Skipping "HelpOpenTicketButton" function: 000001EAB3B6DED0
78.72 NeatMinimap DBG[1]: Adding "BottomRightMiniPanel"
78.72 NeatMinimap DBG[1]: Adding "BottomLeftMiniPanel"
78.72 NeatMinimap DBG[1]: Adding "TopRightMiniPanel"
78.72 NeatMinimap DBG[1]: Adding "TopLeftMiniPanel"
78.72 NeatMinimap DBG[1]: Skipping "TopMiniPanel" function: 000001EAB3B6B9A0
78.72 NeatMinimap DBG[1]: Adding "BottomMiniPanel"
78.72 NeatMinimap DBG[1]: Adding "RightMiniPanel"
78.72 NeatMinimap DBG[1]: Adding "LeftMiniPanel"
78.72 NeatMinimap DBG[1]: Adding "LibDBIcon10_Details"
78.72 NeatMinimap DBG[1]: Adding "LibDBIcon10_BugSack"
78.72 NeatMinimap DBG[1]: Adding "WIM3MinimapButton"
78.72 NeatMinimap DBG[1]: Adding "LibDBIcon10_HealBot"
78.72 NeatMinimap DBG[1]: Adding "MiniMapChallengeMode"
78.72 NeatMinimap DBG[1]: Adding "GuildInstanceDifficulty"
78.72 NeatMinimap DBG[1]: Adding "MiniMapInstanceDifficulty"
78.72 NeatMinimap DBG[1]: Skipping unamed frame/button
78.72 NeatMinimap DBG[1]: Skipping "MMHolder" function: 000001EAB3B6B9A0
78.72 NeatMinimap DBG[1]: Adding "GameTimeFrame"
78.72 NeatMinimap DBG[1]: Skipping "MinimapBackdrop" function: 000001EAB3B6B9A0
78.72 NeatMinimap DBG[1]: Skipping "MiniMapMailFrame" function: 000001EAB3B6B9A0
77.30 NeatMinimap DBG[1]: Hiding buttons - force true
77.30 NeatMinimap DBG[1]: Adding "PixelPerfectAlignminimapButton"
77.30 NeatMinimap DBG[1]: Skipping "NeatMinimapFrame" function: 000001EAB3B6B9A0
77.30 NeatMinimap DBG[1]: Adding "MamaminimapButton"
77.30 NeatMinimap DBG[1]: Adding "LibDBIcon10_AHDBminimapButton"
77.30 NeatMinimap DBG[1]: Skipping "HelpOpenWebTicketButton" function: 000001EAB3B6DED0
77.30 NeatMinimap DBG[1]: Skipping "HelpOpenTicketButton" function: 000001EAB3B6DED0
77.30 NeatMinimap DBG[1]: Adding "BottomRightMiniPanel"
77.30 NeatMinimap DBG[1]: Adding "BottomLeftMiniPanel"
77.30 NeatMinimap DBG[1]: Adding "TopRightMiniPanel"
77.30 NeatMinimap DBG[1]: Adding "TopLeftMiniPanel"
77.30 NeatMinimap DBG[1]: Skipping "TopMiniPanel" function: 000001EAB3B6B9A0
77.30 NeatMinimap DBG[1]: Adding "BottomMiniPanel"
77.30 NeatMinimap DBG[1]: Adding "RightMiniPanel"
77.30 NeatMinimap DBG[1]: Adding "LeftMiniPanel"
77.30 NeatMinimap DBG[1]: Adding "LibDBIcon10_Details"
77.30 NeatMinimap DBG[1]: Adding "LibDBIcon10_BugSack"
77.30 NeatMinimap DBG[1]: Adding "WIM3MinimapButton"
77.30 NeatMinimap DBG[1]: Adding "LibDBIcon10_HealBot"
77.30 NeatMinimap DBG[1]: Adding "MiniMapChallengeMode"
77.30 NeatMinimap DBG[1]: Adding "GuildInstanceDifficulty"
77.30 NeatMinimap DBG[1]: Adding "MiniMapInstanceDifficulty"
77.30 NeatMinimap DBG[1]: Skipping unamed frame/button
77.30 NeatMinimap DBG[1]: Skipping "MMHolder" function: 000001EAB3B6B9A0
77.30 NeatMinimap DBG[1]: Adding "GameTimeFrame"
77.30 NeatMinimap DBG[1]: Skipping "MinimapBackdrop" function: 000001EAB3B6B9A0
77.30 NeatMinimap DBG[1]: Skipping "MiniMapMailFrame" function: 000001EAB3B6B9A0
73.71 NeatMinimap DBG[1]: Hiding buttons - force true
73.71 NeatMinimap DBG[1]: Adding "MamaminimapButton"
73.71 NeatMinimap DBG[1]: Adding "LibDBIcon10_AHDBminimapButton"
73.71 NeatMinimap DBG[1]: Skipping "HelpOpenWebTicketButton" function: 000001EAB3B6DED0
73.71 NeatMinimap DBG[1]: Skipping "HelpOpenTicketButton" function: 000001EAB3B6DED0
73.71 NeatMinimap DBG[1]: Adding "BottomRightMiniPanel"
73.71 NeatMinimap DBG[1]: Adding "BottomLeftMiniPanel"
73.71 NeatMinimap DBG[1]: Adding "TopRightMiniPanel"
73.71 NeatMinimap DBG[1]: Adding "TopLeftMiniPanel"
73.71 NeatMinimap DBG[1]: Skipping "TopMiniPanel" function: 000001EAB3B6B9A0
73.71 NeatMinimap DBG[1]: Adding "BottomMiniPanel"
73.71 NeatMinimap DBG[1]: Adding "RightMiniPanel"
73.71 NeatMinimap DBG[1]: Adding "LeftMiniPanel"
73.71 NeatMinimap DBG[1]: Adding "LibDBIcon10_Details"
73.71 NeatMinimap DBG[1]: Adding "LibDBIcon10_BugSack"
73.71 NeatMinimap DBG[1]: Adding "WIM3MinimapButton"
73.71 NeatMinimap DBG[1]: Adding "LibDBIcon10_HealBot"
73.71 NeatMinimap DBG[1]: Adding "MiniMapChallengeMode"
73.71 NeatMinimap DBG[1]: Adding "GuildInstanceDifficulty"
73.71 NeatMinimap DBG[1]: Adding "MiniMapInstanceDifficulty"
73.71 NeatMinimap DBG[1]: Skipping unamed frame/button
73.71 NeatMinimap DBG[1]: Skipping "MMHolder" function: 000001EAB3B6B9A0
73.71 NeatMinimap DBG[1]: Adding "GameTimeFrame"
73.71 NeatMinimap DBG[1]: Skipping "MinimapBackdrop" function: 000001EAB3B6B9A0
73.71 NeatMinimap DBG[1]: Skipping "MiniMapMailFrame" function: 000001EAB3B6B9A0
73.71 NeatMinimap DBG[1]: Localization not found for "Sets the debug level"
73.71 NeatMinimap DBG[1]: Localization not found for "Debug level"
73.71 NeatMinimap DBG[1]: Localization not found for "Get Information to submit a bug."
73.71 NeatMinimap DBG[1]: Localization not found for "Development, troubleshooting and advanced options:"
73.71 NeatMinimap DBG[1]: Localization not found for "2 ½ s"
73.71 NeatMinimap DBG[1]: Localization not found for "2 s"
73.71 NeatMinimap DBG[1]: Localization not found for "1 ½ s"
73.71 NeatMinimap DBG[1]: Localization not found for "1 s"
73.71 NeatMinimap DBG[1]: Localization not found for "0.5 s"
73.71 NeatMinimap DBG[1]: Localization not found for "none"
73.71 NeatMinimap DBG[1]: Localization not found for "3 sec"
73.71 NeatMinimap DBG[1]: Localization not found for "No delay"
73.71 NeatMinimap DBG[1]: Localization not found for "How long to continue to show the button after mousing out of the map area"
73.71 NeatMinimap DBG[1]: Localization not found for "Hide delay"
73.71 NeatMinimap DBG[1]: Localization not found for "Whether the Blizzard Garrison/Mission/Faction indicator should also be hidden/shown"
73.71 NeatMinimap DBG[1]: Localization not found for "Also hide/show Mission indicator"
73.71 NeatMinimap DBG[1]: Localization not found for "Whether the Blizzard Day/Night indicator should also be hidden/shown"
73.71 NeatMinimap DBG[1]: Localization not found for "Also hide/show Day/Night indicator"
73.71 NeatMinimap DBG[1]: Localization not found for "Whether the Blizzard clock should also be hidden/shown"
73.71 NeatMinimap DBG[1]: Localization not found for "Also hide/show Clock"
73.71 NeatMinimap DBG[1]: Localization not found for "These options let you control the behavior of NeatMinimap"
73.71 NeatMinimap DBG[1]: Localization not found for "Neat Minimap auto hides/shows buttons and clutter as needed"
73.71 NeatMinimap DBG[1]: Localization not found for "NeatMinimap options"
73.71 NeatMinimap DBG[1]: Localization not found for "NeatMinimap"
73.71 NeatMinimap DBG[1]: Creating Options Panel
73.71 NeatMinimap DBG[1]: OnPlayerEnteringWorld "PLAYER_ENTERING_WORLD" , false , true
73.71 NeatMinimap DBG[1]: On ev "UI_SCALE_CHANGED" , nil
73.71 NeatMinimap DBG[1]: On ev "UI_SCALE_CHANGED" , nil
NeatMinimap vX.YY.ZZ by MooreaTv: type /nmm for command list/help.
```

**Comment by mooreatv** [comment]
some more fix coming in 1.4
